src/utility/HCISharedMemTransport.h

```diff
@@ -38,6 +39,8 @@
 /* this CONFIG_DEBUG must be defined for -Og option */
 //#define CONFIG_DEBUG
 
+#define CFG_HW_RCC_SEMID                                        3U
```

Comment:
I think HSEM definitions should be made in stm32duino core, because:
* use of HSEM for RCC should be coherent with other semaphore used within the system and in M0+ core.
* This RCC HSEM should also be used in the rest of the core as soon as RCC register are written.

Reply:
Howver the CFG_HW_RCC_SEMID is used only in the src/utility/HCISharedMemTransport.cpp

Reply:
But as I said, 

> This RCC HSEM should also be used in the rest of the core as soon as RCC register are written.

Reply:
I'm agreed with @ABOSTM .
PR https://github.com/stm32duino/Arduino_Core_STM32/pull/1291 would handle that.
Then we will see if this PR is correct.

Reply:
great

Reply:
This define has been removed. It is not needed as it will be defined by the core thanks #1291